src/pybind/mgr/cephadm/module.py

```diff
@@ -3827,9 +3830,56 @@ def upgrade_resume(self) -> str:
     def upgrade_stop(self) -> str:
         return self.upgrade.upgrade_stop()
 
+    @handle_orch_error
+    def replace_device(self,
+                       hostname: str,
+                       device: str,
+                       clear: bool = False,
+                       yes_i_really_mean_it: bool = False) -> Any:
+        output: str = ''
+
+        self.ceph_volume.lvm_list.get_data(hostname=hostname)
```

Comment:
this usage here sort of reflects what I said in my other comment on the `get_data` function. What we really want is to get the lvm list output, use it for this replace_device workflow and then get rid of it. We don't actually need a persisted `data` field in `self.ceph_volume.lvm_list` for later use.